pkg/server/server.go

```diff
@@ -250,6 +250,9 @@ func (s *secretHandler) sync(_ string, secret *corev1.Secret) (*corev1.Secret, e
 		Webhooks: mutatingWebhooks,
 	}
 	err := s.ensureWebhookConfiguration(validatingConfig, mutatingConfig)
+	if err != nil {
+		logrus.Errorf(err.Error())
```

Comment:
Should we use `logrus.Error` instead of `logrus.Errorf` since we don't have any format string?

Reply:
sure